modules/rosetta/src/main/scala/org/tessellation/rosetta/server/model/network.scala

```diff
@@ -0,0 +1,52 @@
+package org.tessellation.rosetta.server.model
+
+import cats.Show
+import cats.implicits.toBifunctorOps
+
+import derevo.cats.show
+import derevo.derive
+import eu.timepit.refined.auto._
+import eu.timepit.refined.predicates.all.NonEmpty
+import eu.timepit.refined.refineV
+import eu.timepit.refined.types.string.NonEmptyString
+import io.estatico.newtype.macros.newtype
+import io.getquill.MappedEncoding
+
+object network {
+  implicit def showNonEmptyString: Show[NonEmptyString] = s => s.value
+
+  @derive(show)
+  @newtype
+  case class NetworkStatus(value: NonEmptyString)
+
+  object NetworkStatus {
+    val accepted: NetworkStatus = NetworkStatus("Accepted")
+    val reverted: NetworkStatus = NetworkStatus("Reverted")
+    val canceled: NetworkStatus = NetworkStatus("Canceled")
+
+    implicit val quillEncode: MappedEncoding[NetworkStatus, String] =
+      MappedEncoding[NetworkStatus, String](_.value.value)
+
+    implicit val quillDecode: MappedEncoding[String, NetworkStatus] = MappedEncoding[String, NetworkStatus](
+      refineV[NonEmpty].apply[String](_).leftMap(new Throwable(_)) match {
+        case Left(err)    => throw err
+        case Right(value) => NetworkStatus(value)
+      }
+    )
```

Comment:
why you need quill if there is no database yet?

Reply:
This is for when a transaction is written to a database in the implementations of the Rosetta server. Those implementations will be in the `dagL1` module and `quill` will be used there.

Reply:
I forgot to add `quill` to the dependencies for the `rosetta` module. That has been rectified.

Reply:
Can you separate this changes and add database support and quill in later pull request, when it would be needed?

Reply:
because this pull request was meant to add last auto generated classes (as the name says) and right now we have plenty of other stuff in here

Reply:
I'll make the requested changes.

Reply:
The `quill` dependencies and decoders have been removed.